src/components/AddPeopleToTag/AddPeopleToTag.spec.tsx

```
@@ -21,15 +21,69 @@
 import AddPeopleToTag from './AddPeopleToTag';
 import i18n from 'utils/i18nForTest';
+import { USER_TAGS_MEMBERS_TO_ASSIGN_TO } from 'GraphQl/Queries/userTagQueries';
+import { TFunction } from 'i18next';
 import {
   MOCK_EMPTY,
-  MOCK_NULL_FETCH_MORE,
   MOCK_NO_DATA,
   MOCK_NON_ERROR,
-  MOCKS,
-  MOCKS_ERROR,
+  MOCK_NULL_FETCH_MORE,
```

ESLint

[error] 'MOCK_NULL_FETCH_MORE' is defined but never used.

(@typescript-eslint/no-unused-vars)